src/index.js

```diff
@@ -1,8 +1,8 @@
 import * as eases from 'eases-jsnext';
 
 // Polyfills
-const raf = window.requestAnimationFrame || (callback => setTimeout(callback, 1000 / 60));
-const caf = window.cancelAnimationFrame || (id => clearTimeout(id));
+const raf = typeof window !== 'undefined' && window.requestAnimationFrame || (callback => setTimeout(callback, 1000 / 60));
```

Comment:
As you can see the test failing https://travis-ci.org/taisuke-j/easing-animation-frames/jobs/554221728

Reply:
Fixed